src/main/resources/db/migration/V6__create_job_roles_table.sql

```diff
@@ -1,6 +1,9 @@
 CREATE TABLE jobRoles (
     id INT PRIMARY KEY AUTO_INCREMENT NOT NULL,
     role_name VARCHAR(100) NOT NULL,
+    description VARCHAR(2000) NOT NULL,
+    responsibilities VARCHAR(1000) NOT NULL,
+    job_spec VARCHAR(500) NOT NULL,
```

Comment:
don't update existing migrations as they've already been run. instead add a new migration to alter the table to add new columns